plugins/inputs/mock/README.md

```diff
@@ -28,6 +28,9 @@ Below is a sample config to generate one of each of the four types:
 
   ## One or more mock data fields *must* be defined.
   ##
+  ## [[inputs.mock.constant]]
+  ##   name = "rand"
```

Comment:
```suggestion
  ##   name = "constant"
```

Can we call this something other than rand, since the very next example is random?